js/parts-more/PackedBubbleSeries.js

```diff
@@ -12,82 +12,455 @@ import '../parts/Axis.js';
+             */
+            initialPositionRadius: 20,
+            /**
+             * The distance between two bubbles, when the algorithm starts to
+             * treat two bubbles as overlapping. bubblePadding is also the
+             * expected distance between all the bubbles on simulation end.
+             *
+             * @since       7.1.0
+             * @type {Number}
+             * @default 5
+             */
+            bubblePadding: 5,
+            /**
+             * Wheter bubbles should interact with their parentNode to keep them
+             * inside or not.
+             *
+             * @since       7.1.0
+             * @type {boolean}
+             * @default false
+             */
+            parentNodeLimit: false,
+            /**
+             * Wheter series should interact with each other or not. When
+             * parentNodeLimit is set to true, these option should be set to
+             * false to avoid sticking points in wrong series parentNode.
+             *
+             * @since       7.1.0
+             * @type {boolean}
+             * @default true
+             */
+            seriesInteraction: true,
+            /**
+             * In case of splitted series, these option allows user to drag and
+             * drop points between series, for changing point related series.
+             *
+             * @since       7.1.0
+             * @type {boolean}
+             * @default false
+             */
+            dragBetweenSeries: false,
+            /**
+             * Layout algorithm options for Parent Nodes
+             *
+             * @since       7.1.0
+             * @extends plotOptions.networkgraph.layoutAlgorithm
+             * @excluding approximation, attractiveForce, repulsiveForce, theta
+             */
+            parentNodeOptions: {
+                enableSimulation: true,
+                maxIterations: 400,
+                gravitationalConstant: 0.03,
+                maxSpeed: 50,
+                initialPositionRadius: 100,
+                seriesInteraction: true
+            },
+            enableSimulation: true,
+            /**
+             * Type of the algorithm used when positioning bubbles.
+             *
+             * @validvalue  ["packedbubble"]
+             */
+            type: 'packedbubble',
+            /**
+             * Integration type.
+             * Integration determines how forces are applied
+             * on particles. `packedbubble` integration is based on networkgraph
+             * `Verlet` integration, where new position
+             * is based on a previous posittion
+             * without velocity:
+             * `newPosition += previousPosition - newPosition`.
+             *
+             *
+             *
+             * @since       7.1.0
+             * @sample      highcharts/series-networkgraph/forces/
+             * @validvalue  ["packedbubble"]
+             */
+            integration: 'packedbubble',
```

Comment:
Do we want to give users an opportunity to change that? Maybe this option should be `@ ignore`'d?

Reply:
Thanks, changed.